plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/AddToCartWithOptions.php

```diff
@@ -194,6 +214,31 @@ function ( $variation ) {
 				$context['availableVariations'] = $available_variations_data;
 			}
 
+			if ( $product->is_type( 'grouped' ) ) {
+				// Add context for purchasable child products.
+				$context['groupedProductIds'] = array();
+				foreach ( $product->get_children() as $child_product_id ) {
+					$child_product = wc_get_product( $child_product_id );
+					if ( $child_product && $this->is_child_product_purchasable( $child_product ) ) {
+						$context['groupedProductIds'][] = $child_product_id;
+					}
+				}
+
+				// Add quantity context for purchasable child products.
+				$context['quantity'] = array_fill_keys(
+					$context['groupedProductIds'],
+					$default_quantity
```

Comment:
This can be done in a follow-up because it's like that in `trunk`, but I'm not sure why `$default_quantity` defaults to `1` and uses the `woocommerce_add_to_cart_quantity` filter (line 186). I think it should be instead:

```PHP
apply_filters( 'woocommerce_quantity_input_min', $product->get_min_purchase_quantity(), $product )
```

Because we want the default value to be the minimum that can be introduced in the input.